torch_np/tests/numpy_tests/core/test_multiarray.py

```diff
@@ -2003,10 +1847,11 @@ def test_searchsorted_n_elements(self):
         b = a.searchsorted([0, 1, 2], 'right')
         assert_equal(b, [0, 2, 2])
 
+    @pytest.mark.xfail(reason="no .view method")
```

Comment:
Sort of. Did not merge it yet because that PR is tied up with fancy indexing weirness. Can probably cherry-pick the view here and fix the test. There will be several others to fix, so it's either bloat this PR or save for a repo-wide combing though xfails (which is on the todo list anyway).

Reply:
nah it's fine. Let's fix whenever the other PR lands, no rush.